paralink-node

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
 
-pipenv run alembic upgrade head
-pipenv run python3 -m src.__main__ "$@"
+if [ $1 == "node" ]
```

Comment:
If anything it should be vice versa, check for `pytest` first and for `else` statement pass the arguments to the `__main__`. However I touched on the topic in the #26 PR, maybe we should use `tox` for this and leave `./paralink-node` for user interaction only.

Reply:
This was introduced as a _patch_ to allow for testing.  Good call I'll migrate this to `tox`.

Reply:
I would like to have a separation of concerns (user vs dev) but if this is something you perceive is best, then sure, we can leave it as it is.

Reply:
No I agree that we should separate concerns and remove this from from the `paralink-node` file.  What I mean to say is I typically tend to avoid using singletons / module scoped variables (as is the case with the `Config` object) and prefer to inject dependencies.  In any case this is a minor detail and I agree with you that we should favour `tox` as you suggested.